lib/members/member.ex

```diff
@@ -0,0 +1,47 @@
+defmodule GlassFactoryApi.Members.Member do
+  @moduledoc """
+  Defines the Member of organization.
+  """
+
+  alias GlassFactoryApi.Members.Member
+
+  @type t() :: %Member{
+          name: String.t(),
+          email: String.t(),
+          id: integer(),
+          archived: boolean(),
+          capacity: non_neg_integer(),
+          freelancer: boolean(),
+          joined_at: String.t()
+        }
+
+  defstruct [:name, :email, :id, :archived, :capacity, :freelancer, :joined_at]
+
+  @doc """
+  Parse a map into a Member struct
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.to_struct(%{name: "John Doe", id: 100...})
+      %Member{
+        name: "John Doe",
+        id: 100,
+        email: "john.doe@example.org",
+        archived: false,
+        capacity: 6.0,
+        freelancer: false,
+        joined_at: "2019-01-01"
+      }
+  """
+  def to_struct(attrs) do
```

Comment:
We may be able to leverage `Kernel.struct!/2` here

Reply:
I was using `Kernel.struct/2` but I and @mracos discussed about it [here](https://github.com/Cendrao/glass_factory_api/pull/3#discussion_r342187861). 

The response came as a map with string keys, so to make it an atom before using `Kernel.struct/2`, I was doing something like this:

```elixir
for{key, val} <- attrs, into: %{}, do {String.to_atom(key), val}
```

Which bring us the memory fallback of using `String.to_atom/1` with unknown strings.

Do you have another suggestion for this case? 😅